src/views/InputView.tsx

```diff
@@ -232,6 +248,19 @@ class InputView extends React.Component<InputViewProps, any> {
     </div>
     );
   }
+
+  public scrollListener() {
+    const { closePopup } = this.props;
+    closePopup();
+  }
+
+  private setScrollListener() {
+    window.addEventListener('scroll', this.scrollListener.bind(this));
+  }
+
+  private unsetScrollListener() {
+    window.removeEventListener('scroll', this.scrollListener.bind(this));
```

Comment:
Here you actually do not remove 'scroll' listener because `this.scrollListener.bind` creates new function instance.